source/common/http/http3/conn_pool.cc

```diff
@@ -31,6 +31,16 @@ ActiveClient::ActiveClient(Envoy::Http::HttpConnPoolImplBase& parent,
                                   parent.host()->cluster().stats().upstream_cx_http3_total_, data) {
 }
 
+void ActiveClient::onEnlisted() {
+  // HTTP/3 codec hasn't tried to connect yet.
+  MultiplexedActiveClientBase::onEnlisted();
+  ASSERT(codec_client_);
+  if (dynamic_cast<CodecClientProd*>(codec_client_.get()) == nullptr) {
+    // Hasn't called connect() yet.
+    codec_client_->connect();
```

Comment:
Presumably this only happen in the case where the runtime flag is false? We could ASSERT() this here, but maybe that's not worth it?

Reply:
Done